From: Michael Tokarev <mjt@tls.msk.ru>
Date: Fri, 01 Oct 2021 09:39:44 +0300
Subject: disable running non-existing ./doxygen.sh in meson.build

For some reason upstream's meson.build ends up with run_target( ./doxygen.sh ),
but doxygen.sh is not in the upstream source tarball and is not being generated
either.  This is catched by meson 0.59, but not, eg, 0.56, for some weird reason,
so this line apparently went unnoticed.  Comment it out for now.

--- a/meson.build
+++ b/meson.build
@@ -232 +232 @@ configure_file(output : 'config.h',
-run_target('doxy', command : './doxygen.sh')
+#run_target('doxy', command : './doxygen.sh')
